pkg/replicas/putter.go

```diff
@@ -31,9 +31,9 @@ func NewPutter(p storage.Putter) storage.Putter {
 // Put makes the getter satisfy the storage.Getter interface
 func (p *putter) Put(ctx context.Context, ch swarm.Chunk) (err error) {
 	rlevel := redundancy.GetLevelFromContext(ctx)
-	errs := []error{p.putter.Put(ctx, ch)}
+	errs := []error{}
```

Comment:
Ah, so THIS is where the redundant put of the root chunk came from?  I was wondering!  Do you mind my asking (for my own education) where the root chunk actually gets put for rlevel==0?

Reply:
it is done by the hashtrie module which is in a chunk pipeline process that has a `ChainWrite` function which does the job.